{"cve": {"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2009-0314", "ASSIGNER": "cve@mitre.org"}, "problemtype": {"problemtype_data": [{"description": [{"lang": "en", "value": "CWE-426"}]}]}, "references": {"reference_data": [{"url": "http://bugzilla.gnome.org/show_bug.cgi?id=569214", "name": "http://bugzilla.gnome.org/show_bug.cgi?id=569214", "refsource": "", "tags": ["Exploit", "Issue Tracking", "Patch", "Vendor Advisory"]}, {"url": "http://bugzilla.gnome.org/show_bug.cgi?id=569214", "name": "http://bugzilla.gnome.org/show_bug.cgi?id=569214", "refsource": "", "tags": ["Exploit", "Issue Tracking", "Patch", "Vendor Advisory"]}, {"url": "http://secunia.com/advisories/33759", "name": "33759", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "http://secunia.com/advisories/33759", "name": "33759", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "http://secunia.com/advisories/33769", "name": "33769", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "http://secunia.com/advisories/33769", "name": "33769", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "http://secunia.com/advisories/34522", "name": "34522", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "http://secunia.com/advisories/34522", "name": "34522", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "http://security.gentoo.org/glsa/glsa-200903-41.xml", "name": "GLSA-200903-41", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "http://security.gentoo.org/glsa/glsa-200903-41.xml", "name": "GLSA-200903-41", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "http://www.mandriva.com/security/advisories?name=MDVSA-2009:039", "name": "MDVSA-2009:039", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "http://www.mandriva.com/security/advisories?name=MDVSA-2009:039", "name": "MDVSA-2009:039", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "http://www.openwall.com/lists/oss-security/2009/01/26/2", "name": "[oss-security] 20090126 CVE request -- Python < 2.6 PySys_SetArgv issues (epiphany, csound, dia, eog, gedit, xchat, vim, nautilus-python, Gnumeric)", "refsource": "", "tags": ["Mailing List"]}, {"url": "http://www.openwall.com/lists/oss-security/2009/01/26/2", "name": "[oss-security] 20090126 CVE request -- Python < 2.6 PySys_SetArgv issues (epiphany, csound, dia, eog, gedit, xchat, vim, nautilus-python, Gnumeric)", "refsource": "", "tags": ["Mailing List"]}, {"url": "http://www.securityfocus.com/bid/33445", "name": "33445", "refsource": "", "tags": ["Third Party Advisory", "VDB Entry"]}, {"url": "http://www.securityfocus.com/bid/33445", "name": "33445", "refsource": "", "tags": ["Third Party Advisory", "VDB Entry"]}, {"url": "https://bugzilla.redhat.com/show_bug.cgi?id=481556", "name": "https://bugzilla.redhat.com/show_bug.cgi?id=481556", "refsource": "", "tags": ["Exploit", "Issue Tracking", "Patch", "Third Party Advisory"]}, {"url": "https://bugzilla.redhat.com/show_bug.cgi?id=481556", "name": "https://bugzilla.redhat.com/show_bug.cgi?id=481556", "refsource": "", "tags": ["Exploit", "Issue Tracking", "Patch", "Third Party Advisory"]}, {"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/48271", "name": "gedit-pysyssetargv-privilege-escalation(48271)", "refsource": "", "tags": ["Third Party Advisory", "VDB Entry"]}, {"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/48271", "name": "gedit-pysyssetargv-privilege-escalation(48271)", "refsource": "", "tags": ["Third Party Advisory", "VDB Entry"]}, {"url": "https://www.redhat.com/archives/fedora-package-announce/2009-January/msg01195.html", "name": "FEDORA-2009-1189", "refsource": "", "tags": ["Third Party Advisory"]}, {"url": "https://www.redhat.com/archives/fedora-package-announce/2009-January/msg01195.html", "name": "FEDORA-2009-1189", "refsource": "", "tags": ["Third Party Advisory"]}]}, "description": {"description_data": [{"lang": "en", "value": "Untrusted search path vulnerability in the Python module in gedit allows local users to execute arbitrary code via a Trojan horse Python file in the current working directory, related to a vulnerability in the PySys_SetArgv function (CVE-2008-5983)."}]}}, "configurations": {"CVE_data_version": "4.0", "nodes": [{"operator": "OR", "children": [], "cpe_match": [{"vulnerable": true, "cpe23Uri": "cpe:2.3:a:gnome:libpeas:*:*:*:*:*:*:*:*", "versionEndExcluding": "0.5.3", "cpe_name": []}]}, {"operator": "OR", "children": [], "cpe_match": [{"vulnerable": true, "cpe23Uri": "cpe:2.3:o:fedoraproject:fedora:9:*:*:*:*:*:*:*", "cpe_name": []}]}]}, "impact": {"baseMetricV2": {"cvssV2": {"version": "2.0", "vectorString": "AV:L/AC:M/Au:N/C:C/I:C/A:C", "accessVector": "LOCAL", "accessComplexity": "MEDIUM", "authentication": "NONE", "confidentialityImpact": "COMPLETE", "integrityImpact": "COMPLETE", "availabilityImpact": "COMPLETE", "baseScore": 6.9}, "severity": "MEDIUM", "exploitabilityScore": 3.4, "impactScore": 10.0, "acInsufInfo": false, "obtainAllPrivilege": true, "obtainUserPrivilege": false, "obtainOtherPrivilege": false, "userInteractionRequired": true}}, "publishedDate": "2009-01-28T11:30Z", "lastModifiedDate": "2024-11-21T00:59Z"}